ci/publish.sh

```diff
@@ -8,6 +8,8 @@ BASEDIR="$( dirname "${THISDIR}" )"
 source "${BASEDIR}/ci/shared/_app_helper.sh"
 source "${BASEDIR}/ci/shared/_docker_helper.sh"
 
+docker-compose build --no-cache
```

Comment:
should this be one this line here? https://github.com/3amigos-dev/pipenv-all/blob/2135d634a057d66746c73a8cf26de506a05eca99/ci/shared/_docker_helper.sh#L5

Reply:
thx have removed that build